node/forest_libp2p/src/service.rs

```diff
@@ -644,7 +658,13 @@ async fn handle_forest_behaviour_event<DB, P: StoreParams>(
                 );
             }
             Err(err) => {
-                warn!("{err}:{}", ping_event.peer.to_base58());
+                let err = err.to_string();
```

Comment:
is this line needed?

Reply:
I haven't confirmed the I guess without this line, `warn!` will make an extra allocation by calling `err.to_string` in `Display` . Not a big deal tho I can change it if you prefer fewer lines

Reply:
No, it's okay; I didn't notice the 2nd usage.